### PR TITLE
update kir-dev courses for 2022 spring

### PR DESCRIPTION
**Pull request by SepsiLaszlo**

### CODE REVIEW

File: src/data/courses/kir-dev/tanfolyam.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Kir-Dev Tanfolyam
+society: Kir-Dev
+occasions:
+  - instructors:
+      - Bucsy Benjámin, Piller Trisztán, Berente Bálint, Kovács Bertalan, Sepsi László
```
Comment (triszt4n) [inline src/data/courses/kir-dev/tanfolyam.md]: Maybe list the names alphabetically. That way it is correct for everyone :)

File: src/data/courses/kir-dev/tanfolyam.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
```
Comment (triszt4n) [inline src/data/courses/kir-dev/tanfolyam.md:9]:
```suggestion
moreInfoURL: https://kir-dev.sch.bme.hu/courses
```

File: src/data/courses/kir-dev/tanfolyam.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+tags:
+  - web
+  - programozás
+---
+
+Szeretnél innovatív weboldalakat készíteni villámgyorsan? Gyere el a tanfolyamainkra, ahol a webfejlesztés alapjainak elsajátítása után, választhatsz a NodeJS és a Ruby on Rails back-end technológiák megismerése közül. Végül a React front-end keretrendszer rejtelmeibe is betekintést nyerhetsz.
+
+A részleteket a weboldalunkon találhatod meg:
+
+https://kir-dev.sch.bme.hu/courses
```
Comment (triszt4n) [inline src/data/courses/kir-dev/tanfolyam.md]: It is not safe to insert urls without the correct markdown syntax:
```suggestion
A részleteket [a weboldalunkon](https://kir-dev.sch.bme.hu/courses) találhatod meg.
```

**Review by triszt4n — CHANGES_REQUESTED** [review]
Very good idea, I like it, but I've left some issues, let's talk about them!

**Comment by triszt4n** [comment]
Just some extra in this scope:

https://github.com/simonyiszk/tanfolyam-web/blob/e87cfa2a1ce08d89fe46250dc33e76b49b8cae60/src/data/societies.yaml#L19

This line has our link in `http`

**Comment by SepsiLaszlo** [comment]
@triszt4n thank you for the suggestions, I made the requested changes!
Also added the linkt to our application form.

**Review by triszt4n — APPROVED** [review]

**Review by DannySS123 — APPROVED** [review]

**Comment by triszt4n** [comment]
@SepsiLaszlo This is still a Draft PR, you can switch it to a normal PR, and I think we can merge this if you think the work is done here.

**Pull request merged**